oll_sc/cli.py

```diff
@@ -11,6 +12,27 @@ def oll_sc():
   """oll-sc tool CLI"""
 
 
+@oll_sc.command()
+@click.argument('key_id', type=int)
+@click.argument('pin')
+@click.option('--output-path', '-o', type=click.Path(), default=None,
+              help='The output file path to write public key pem to.')
+def public_key(key_id, pin, output_path=None):
+  """Extract public key from smart card in PEM format."""
+  try:
+    pub_key_pem_bytes = sc_export_pub_key_pem((key_id,), pin)
+    pub_key_pem = pub_key_pem_bytes.decode('utf-8')
```

Comment:
I would add this to the api. So, a function which public key pem as a string.

Reply:
The reason I left bytes is that we don't have to worry about line endings when, for e.g., writing to a file. I think we are fine here, since conversion to a string could be done in one line:

`sc_export_pub_key_pem((key_id,), pin).decode()`

Reply:
I was thinking about adding another function which returns a string (wraps the two calls). That sounds quite convenient to me.